tailwind.config.js

```diff
@@ -2,7 +2,57 @@
 module.exports = {
   content: ['./src/**/*.{html,ts}', 'src/**/*.{css,scss}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        accent: '#3545E9',
+        warning: '#FEFD54',
+        success: '#63C995',
+        error: '#E23D69',
+        primary: '#222328',
+        secondary: '#969AB0',
+        bright: '#FFFFFF',
+        muted: '#7C9CBF',
+        shade: '#DBE2EA',
+      },
+      fontFamily: {
+        'ibm-plex-sans-regular': ["IBMPlexSans-Regular"],
+        'ibm-plex-sans-semi-bold': ["IBMPlexSans-SemiBold"]
+      },
+      fontSize: {
+        x1: '72px',
+        x2: '64px',
+        x3: '56px',
+        x4: '34px',
+        x5: '28px',
+        x6: '24px',
+        paragraphX1: '16px',
+        paragraphX2: '14px',
+        label: '12px'
+      },
+      lineHeight: {
+        x1: '86px',
+        x2: '76px',
+        x3: '68px',
+        x4: '44px',
+        x5: '36px',
+        x6: '28px',
+        paragraphX1: '22px',
+        paragraphX2: '20px',
+        label: '18px'
+      },
+      borderRadius: {
+        DEFAULT: '4px',
+      },
+      borderWidth: {
+        DEFAULT: '2px',
```

Comment:
no need for this rule, there already is a tailwind class - border
https://tailwindcss.com/docs/border-width

Reply:
It's default value is 1px, but you can use border-2 for 2px